index.js

```diff
@@ -447,13 +456,26 @@ module.exports = function (log, indexesPath) {
           else saveIndex(op.data.indexName, index)
         }
 
+        waitingIndexUpdate[waitingKey].forEach((cb) => cb())
+        delete waitingIndexUpdate[waitingKey]
+
         cb()
       },
     })
   }
 
+  // concurrent index create
+  const waitingIndexCreate = {}
+
   function createIndexes(opsMissingIndexes, cb) {
     const newIndexes = {}
+
+    const waitingKey = opsMissingIndexes.join()
```

Comment:
I think this should be `opsMissingIndexes.map(op => op.data.indexName).join()`, right? Otherwise we get `[Object],[Object]` strings

Reply:
and maybe with a separator, like `join('|')` to make sure we don't do `value_content_x` + `value_content_y` = `value_content_xvalue_content_y`